common/setups/rasr/util/nn.py

```diff
@@ -148,13 +148,15 @@ def get_crp(self, **kwargs):
 class OggZipHdfDataInput:
     def __init__(
         self,
-        oggzip_files: tk.Path,
+        oggzip_files: List[tk.Path],
         alignments: tk.Path,
```

Comment:
I was just wondering if this should also be Optional?

Reply:
`alignments` you mean, right? If we make this optional, the question is why it should still be the oggzipHDFdatainput class as it's only using ogg zip. We should also not put the `HDFDataset` into the data dict in this case becase we don't have any argument for `files`, and then the `data_map` would also need to be reduced. I think it would be easier to have a `OggZipDataInput` for that case.

Reply:
Mabye then something like: `OggZipHdfDataInput(OggZipDataInput)` ^^